taskboot/cli.py

```diff
@@ -93,7 +93,12 @@ def main():
         help='Path to docker-compose.yml to use',
         default='docker-compose.yml',
     )
-    compose.add_argument('--registry', type=str, help='Docker registry to use in images tags')
+    compose.add_argument(
+        '--registry',
+        type=str,
+        default=os.environ.get('REGISTRY', 'registry.hub.docker.com'),
```

Comment:
Maybe use the same default in the build --registry argument?

Reply:
good point

Reply:
Sure, making a commit right now